plugins/wallabag.koplugin/main.lua

```diff
@@ -178,6 +182,14 @@ function Wallabag:addToMainMenu(menu_items)
                             self:saveSettings()
                         end,
                     },
+                    {
+                        text = _("Archive instead of delete articles"),
```

Comment:
Is that "mark as read"? Because if so, I think something like `mark as read instead of deleting` would be clearer.
```suggestion
                        text = _("Archive instead of deleting"),
```

Reply:
Yes, it is. You're right, it is essentially the `Mark as read` functionality. It seems to be called `archived` in the Wallabag API.

Reply:
It's just not a term I'm familiar with from the Wallabag website, and if I saw it in the API docs I forgot. :-)